ghc-typelits-knownnat.cabal

```diff
@@ -87,7 +87,7 @@ library
                        UndecidableInstances
                        ViewPatterns
   build-depends:       base                      >= 4.9      && <5,
-                       ghc                       >= 8.0.1    && <9.10,
+                       ghc                       >= 8.0.1    && <9.12,
```

Comment:
Not a fan of these version bumps without actually testing it against GHC 9.10 in CI.

Reply:
I have tested locally but I would be fine with dropping them for now.